includes/ingest.form.inc

```diff
@@ -922,6 +922,20 @@ function islandora_ingest_form_get_steps(array &$form_state) {
 }
 
 /**
+ * Implements hook_form_alter().
+ */
+function islandora_form_islandora_ingest_form_alter(&$form, &$form_state) {
+  foreach ($form_state['islandora']['shared_storage']['models'] as $key => $cmodel) {
+    $fedora_cmodel = islandora_object_load($cmodel);
+    $cmodel_label = $fedora_cmodel->{'label'};
```

Comment:
Travis is correctly complaining here: 
 islandora_form_islandora_ingest_form_alter() failed: Trying to  [error]
get property of non-object in
/home/travis/build/Islandora/islandora/includes/ingest.form.inc on 930 since it's very possible in current Islandora to have a CMODEL assigned that really does not exist! (like test:testcmodel)

You have many options there like, testing if $fedora_cmodel) before trying to access the property, change the tests! etc. 
Why `->{'label'}` instead of directly `->label`?